model/bundle_info.go

```diff
@@ -11,6 +11,8 @@ type BundleInfo struct {
 	Manifest      *Manifest
 	ManifestPath  string
 	ManifestError error
+
+	HealthCheckFails int
```

Comment:
Not stateless here. I believe we need to store the health check failure elsewhere so it will work in a distributed context.

Reply:
Certainly the bundle info should be read-only.

Reply:
Right, this is a placeholder per plugin until it is decided where to store this info. It definitely doesn't feel right to store this here.

Reply:
Yeah I don't think we need to worry about a distributed context here, the health check is for the plugin running on a specific app server so only that app server cares